### DISCUSSION TITLE
Definition of umean and uumean

### DISCUSSION THREAD

**Discussion by Nakhchi** [xcompact3d/Incompact3d#188] [Q&A] [answered]
Hi,

When I run wind turbine simulations, some mean value outputs are created. 

Does `uumean` mean $\overline{u'u'}$ ? 
Does `umean` mean $\overline{u}$ ? 

Thank you for your help.

**Comment by slaizet** [comment, marked as answer]
Hi, 
umean is indeed `<u>` and uumean is `<uu>`, with `u` defined as `u=<u>+u'`
To get `<u'u'>` you just need to do `<u'u'>=<uu>-<u><u>`, because `uu=(<u>+u')^2`
Best,
Sylvain

**Reply by gauravity30** [reply]
Is umean and uumean time-averaged or space averaged ? I read the statistics.f90 and I feel it is time-averaged. Can you please confirm ?

**Reply by slaizet** [reply]
time-averaged.
Best,
Sylvain
